### PR TITLE
Better examples, fix formatting

### PR DESCRIPTION
**Pull request by tonypiper**
use better examples - user id 404 might be mistaken for an http status code, and 'song:played' is a more realistic action (to most people).

Also fix formatting (this file is using restructured text, not markdown).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -70,12 +63,12 @@ Get Bitter:
 
     $bitter = $this->container->get('rezzza.bitter');
 
-Mark user 404 as active and has been kicked by Chuck Norris:
```
Comment (jeremyFreeAgent) [inline README.rst:73]: hum... 404 is one of my favorite number...

File: README.rst [hunk 2]
```diff
@@ -70,12 +63,12 @@ Get Bitter:
...
+Mark user 123 as active and has played a song:
 
 .. code-block:: php
 
-    $bitter->mark('active', 404);
-    $bitter->mark('kicked_by_chuck_norris', 404);
```
Comment (jeremyFreeAgent) [inline README.rst:78]: Chuck Norris won't be happy for that ;)

**Comment by jeremyFreeAgent** [comment]
Thanks @tonypiper :+1:

**Comment by stephpy** [comment]
Thanks :)

**Pull request merged**